bridges/primitives/runtime/src/lib.rs

```diff
@@ -76,8 +76,8 @@ pub const KUSAMA_CHAIN_ID: ChainId = *b"ksma";
 /// Westend chain id.
 pub const WESTEND_CHAIN_ID: ChainId = *b"wend";
 
-/// Westend chain id.
-pub const WESTMINT_CHAIN_ID: ChainId = *b"wmnt";
+/// AssetHubWestend chain id.
+pub const ASSET_HUB_WESTEND_CHAIN_ID: ChainId = *b"ahwe";
```

Comment:
I presume this is not used anywhere live, and has no consequence to change?

Reply:
Correct, https://github.com/paritytech/cumulus/pull/2633#discussion_r1205620955